index.html

```diff
@@ -0,0 +1,191 @@
+					<div class="block-info">
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+			<a href="#">
+				<div class="block-pizza">
+					<img src="img/4.png" alt="Pizza foto">
+					<div class="pizza-title">SALIAMI</div>
+					<div class="block-info">
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+			<a href="#">
+				<div class="block-pizza">
+					<img src="img/5.png" alt="Pizza foto">
+					<div class="pizza-title">SALSISHA</div>
+					<div class="block-info">
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+			<a href="#">
+				<div class="block-pizza">
+					<img src="img/6.png" alt="Pizza foto">
+					<div class="pizza-title">WITH CHIKEN AND CHEADER</div>
+					<div class="block-info">
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+			<a href="#">
+				<div class="block-pizza">
+					<img src="img/7.png" alt="Pizza foto">
+					<div class="pizza-title">PROSHOOTO</div>
+					<div class="block-info">
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+			<a href="#">
+				<div class="block-pizza">
+					<img src="img/8.png" alt="Pizza foto">
+					<div class="pizza-title">MARGARITTA</div>
+					<div class="block-info">
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+			<a href="#">
+				<div class="block-pizza">
+					<img src="img/9.png" alt="Pizza foto">
+					<div class="pizza-title">CALSONE</div>
+					<div class="block-info">
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+		</div>
+	</content>
+
+	<footer>
```

Comment:
If you would have only one pizza in queue, for example, and the screen size is large, then your footer would jump from the bottom from the page
Like this:
![default](https://user-images.githubusercontent.com/31585545/35692129-2d9e83c8-0783-11e8-96d8-922307d384c9.jpg)
Lempiy pointed me the same problem. One of the best way to make sticky footer is by `display: flex`

Reply:
Here is an example:

- [code](https://github.com/viktor-yakubiv/kottans-frontend/blob/master/memory-pair-game/index.css#L39-L63) (see selected lines)
- [preview](https://viktor-yakubiv.github.io/kottans-frontend/memory-pair-game/)